networkx/algorithms/coloring/greedy_coloring.py

```diff
@@ -10,14 +10,16 @@
 """
 import itertools
 import random
+from collections import defaultdict
 
 import networkx as nx
 from networkx.utils import arbitrary_element
 from . import greedy_coloring_with_interchange as _interchange
 
 __author__ = "\n".join(["Christian Olsson <chro@itu.dk>",
                         "Jan Aagaard Meier <jmei@itu.dk>",
-                        "Henrik Haugbølle <hhau@itu.dk>"])
+                        "Henrik Haugbølle <hhau@itu.dk>",
+                        "Arya McCarthy <admccarthy@smu.edu>"])
```

Comment:
More precisely, see [the wiki](https://github.com/networkx/networkx/wiki) for brief description of the way to include author information (as a comment instead of the variable __author__).  You should add your name to the list... but could you change the way the list is handled?